app/views/leagues/show.html.haml

```diff
@@ -1,3 +1,7 @@
+- content_for :league_header_image do
+  - if @league.league_header_image?
```

Comment:
ich glaub das is immer true, weil da n uploader zurückkommt selbst wenn du kein bild hast, oder erinner ich mich falsch? kann auch bei ner anderen uploader lib gewesen sein.